src/components/input.ts

```diff
@@ -45,18 +47,18 @@ const styles = {
     fontWeight: 400,
     padding: '0.5rem 1rem',
     cursor: 'text',
-  },
+  } satisfies React.CSSProperties,
   validationError: {
     color: colors.danger(),
     fontSize: 11,
     fontWeight: 600,
     textTransform: 'uppercase',
     marginLeft: '1rem',
     marginTop: '0.5rem',
-  },
+  } satisfies React.CSSProperties,
 };
 
-export const withDebouncedChange = (WrappedComponent) => {
+export const withDebouncedChange = (WrappedComponent: React.FC<any>) => {
```

Comment:
a later pass on these types here and below can improve beyond props type "any" to use generic <P> props passthru type-flow.